frontend/src/components/InputField/index.tsx

```diff
@@ -5,23 +5,26 @@ import { Link } from 'react-router-dom';
 import styles from './styles.module.sass';
 
 interface IProps {
-    label: string;
-    name: string;
-    type: string;
-    placeholder?: string;
-    link?: string;
-    linkDescription?: string;
-    linkClassName?: string;
+  label: string;
+  name: string;
+  type: string;
+  placeholder?: string;
+  link?: string;
+  linkDescription?: string;
+  linkClassName?: string;
+  labelClassName?: boolean
 }
 
-const InputField :React.FC<IProps> = ({
+const InputField: React.FC<IProps> = ({
   label,
   link,
   linkDescription,
   linkClassName,
+  labelClassName,
   ...props
 }: IProps) => {
   const [field, meta] = useField(props);
+  const labelClass = labelClassName;
```

Comment:
Can we use `labelClassName` in places where we use `labelClass`? Then we won't need to reassign this variable